src/CurrentSelection/CurrentSelection.tsx

```diff
@@ -5,13 +5,13 @@ import Player from '../Player/Player';
 import { CurrentUser } from '../Teams/Teams';
 
 interface CurrentSelectionProps {
-  team: Array<String>;
+  team: Array<string>;
```

Comment:
This is fine, but at WT you'll almost always see `string[]` instead of `Array<string>`

Reply:
I don't know about the performance or whatever implications of one vs the other, but `string[]` is definitely clearer